Let it build with poppler-0.73.0.

Funny enough, neither any of the types defined in goo/gtypes.h nor the
include of poppler-config.h was needed.

Index: src/ApvlvPdf.cc
--- src/ApvlvPdf.cc.orig
+++ src/ApvlvPdf.cc
@@ -29,7 +29,6 @@
 #include "ApvlvPdf.h"
 
 #ifndef POPPLER_WITH_GDK
-#include <goo/gtypes.h>
 
 static void
 copy_cairo_surface_to_pixbuf (cairo_surface_t *surface,
